### ISSUE TITLE
When using the "colloquial" function, there is an error

### ISSUE THREAD

**Issue by alexsanderkhitev** [malcommac/SwiftDate#419]
Hello! When using the function DateInRegionFormatter colloquial there is an error, today's date is April 20, and I set the time to April 24 at 7 am and it shows that this event will be in 16 hours. How can I fix this? Thanks

My date Apr 24, 2017, 7:00:00 AM GMT+4 
I got this result "in 16 hours" Optional("at 07:00")

My code

```
    let formatter = DateInRegionFormatter()
        formatter.allowedComponents = [.day, .year, .month, .hour, .minute]
        do {
            let result = try formatter.colloquial(from: cardDate, to: nowRegionDate)
            debugPrint(result.colloquial, result.time)
            return result.colloquial
        } catch {
            debugPrint(error.localizedDescription, error)
        }
```

**Comment by RolandasRazma** [comment]
same as https://github.com/malcommac/SwiftDate/issues/417

**Comment by malcommac** [comment]
Just fixed in 4.1.2 alongside new unit tests.
Thank you!

**Comment by alexsanderkhitev** [comment]
@malcommac thank you so much!
